src/instances.ts

```diff
@@ -314,10 +314,16 @@ function addAssetHooks(loader: WebpackLoaderContext, instance: TSInstance) {
   );
 
   const makeAssetsCallback = (compilation: webpack.Compilation) => {
-    compilation.hooks.afterProcessAssets.tap('ts-loader', () =>
-      cachedMakeAfterCompile(compilation, () => {
-        return null;
-      })
+    compilation.hooks.processAssets.tap(
+      {
+        name: 'ts-loader',
+        stage: webpack.Compilation.PROCESS_ASSETS_STAGE_ADDITIONAL,
```

Comment:
What's the significance of the specified stage @JonWallsten?

Reply:
"add additional assets to the compilation." I guess?

Reply:
This is just the preferred way of doing it in Webpack 5. @alexander-akait metioned it in the beginning of the PR. It shouldn't affect anything at all. It's just more future proof.

Reply:
Yep, it is right usage, we can implement cache here in future